waku/v2/peermanager/peer_manager.go

```diff
@@ -0,0 +1,121 @@
+package peermanager
+
+import (
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/libp2p/go-libp2p/core/network"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/core/protocol"
+	wps "github.com/waku-org/go-waku/waku/v2/peerstore"
+	"golang.org/x/exp/slices"
+
+	"go.uber.org/zap"
+)
+
+// TODO: Move all the protocol IDs to a common location.
+// WakuRelayIDv200 is protocol ID for Waku v2 relay protocol
+const WakuRelayIDv200 = protocol.ID("/vac/waku/relay/2.0.0")
```

Comment:
this constant can be obtained from `relay` package

Reply:
If i do that, it was causing some circular dependency. 
I think we will have to move all protocol constants to a common file so that they can be imported wherever we want.